cockatrice/src/decklistmodel.cpp

```diff
@@ -72,10 +72,13 @@ int DeckListModel::rowCount(const QModelIndex &parent) const
 
 int DeckListModel::columnCount(const QModelIndex &/*parent*/) const
 {
+    /* Prices Disabled
     if (settingsCache->getPriceTagFeature())
         return 3;
     else
         return 2;
+    */
+    return 2;
```

Comment:
I wouldn't bother commenting this; leave the behavior, just change `settingsCache.getPriceTagFeature()` to hardcode `return false` with a comment linking to the ticket.

This will make future updates easier when we do have a price source again